script/deploy

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Blue-green deployment script. Usage:
+#
+#   ./script/deploy <appname>
+#
+# Based on
+#   http://www.cloudfoundry.rocks/blue-green-deployment-with-cloudfoundry/
+#   http://docs.pivotal.io/pivotalcf/devguide/deploy-apps/blue-green.html
+#   https://github.com/dlapiduz/step-cloud-foundry-deploy/blob/master/run.sh
+
+set -e
+set -x
+
+PRIMARY=$1
```

Comment:
I was going to agree at this point but since the point of the script is to delete the primary app and make the secondary primary then it kind of makes sense.